test/serde/msgpack/test_msgpack_serde_full.py

```diff
@@ -98,6 +100,7 @@
 samples[SerializableDummyClass] = make_serializable_dummy_class
 
 
+@pytest.mark.xfail("os_name == 'nt'")
```

Comment:
Could you add the `reason` attribute to these?

Reply:
Of course, added attribute